modules/telco-observability-workload-monitoring.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/observability/telco-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-observability-workload-monitoring_{context}"]
+= Workload monitoring
+
+By default, {product-title} does not collect metrics for application workloads. You can configure a cluster to collect workload metrics.
+
+.Prerequisites
+
+* You have defined endpoints to gather workload metrics on the cluster.
+
+.Procedure
+
+. Create a `ConfigMap` CR and save it as `monitoringConfigMap.yaml`, as in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true <1>
+----
+<1> Set to `true` to enable workload monitoring.
+
+. Apply the `ConfigMap` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f monitoringConfigMap.yaml
+----
+
+. Create a `ServiceMonitor` CR, and save it as `monitoringServiceMonitor.yaml`, as in the following example:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: ui
+  name: myapp
+  namespace: myns
+spec:
+  endpoints: <1>
+  - interval: 30s
+    port: ui-http
+    scheme: http
+    path: /healthz
+  selector:
+    matchLabels:
+      app: ui
+----
+<1> Use endpoints to define workload metrics. 
+
+. Apply the `ServiceMonitor` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f monitoringServiceMonitor.yaml
+----
+
+Prometheus scrapes the path `/metrics` by default, however you can define a custom path in the spec. 
```

Comment:
```suggestion
Prometheus scrapes the path `/metrics` by default, however you can define a custom path in the `spec.` fields. 
```

Reply:
Thanks, I've made this a bit clearer, I think.